requirements-test.txt

```diff
@@ -0,0 +1,5 @@
+flake8<2.7.0,>=2.6.0
```

Comment:
Can we switch use flake8 > 3.0.0, aka what we do in StackStorm/st2 at the moment?

Reply:
The reason is that there are some new and improved checks in the latest version.

Reply:
Done!

Reply:
flake8==3.7.7 copied from st2